Compilation fails on armv7

Hi!

Compiling this crate using stable on an armv7 (stable-armv7-unknown-linux-gnueabihf) fails with the following error:

```
error[E0308]: mismatched types
   --> /root/.cargo/git/checkouts/lmdb-rs-82a4129c4785dceb/37b241c/src/error.rs:119:53
    |
119 |             str::from_utf8_unchecked(CStr::from_ptr(err).to_bytes())
    |                                                     ^^^ expected u8, found i8
    |
    = note: expected type `*const u8`
               found type `*const i8`
    = help: here are some functions which might fulfill your needs:
            - .offset(...)
            - .wrapping_offset(...)

```

--- a/src/error.rs
+++ b/src/error.rs
@@ -1,6 +1,7 @@
 use libc::c_int;
 use std::error::Error as StdError;
 use std::ffi::CStr;
+use std::os::raw::c_char;
 use std::{fmt, result, str};
 
 use ffi;
@@ -115,7 +116,7 @@ impl StdError for Error {
     fn description(&self) -> &str {
         unsafe {
             // This is safe since the error messages returned from mdb_strerror are static.
-            let err: *const i8 = ffi::mdb_strerror(self.to_err_code()) as *const i8;
+            let err: *const c_char = ffi::mdb_strerror(self.to_err_code()) as *const c_char;
             str::from_utf8_unchecked(CStr::from_ptr(err).to_bytes())
         }
     }

